src/bun.zig

```diff
@@ -8,18 +8,18 @@ const bun = @This();
 
 pub const Environment = @import("./env.zig");
 
-pub const use_mimalloc = true;
+pub const use_mimalloc = !Environment.enable_asan;
 
 pub const default_allocator: std.mem.Allocator = if (use_mimalloc)
     allocators.c_allocator
 else
-    std.heap.c_allocator;
+    std.heap.raw_c_allocator;
```

Comment:
why raw_c_allocator instead of c_allocator? is it better with asan or is there a different reason?

Reply:
on linux and macos c_allocator uses posix_memalign() and i hadnt looked into libc source to see if that makes its way to malloc. raw_c_allocator always uses malloc.